creusot/src/backend/clone_map.rs

```diff
@@ -469,7 +469,7 @@ pub enum Kind {
 }
 
 impl Kind {
-    fn ident(&self) -> Ident {
+    pub(crate) fn ident(&self) -> Ident {
```

Comment:
Does this still need to be `pub(crate)` now that you've removed the hacks in `program.rs`?

Reply:
Right, thanks!